server/api/email_handler.py

```diff
@@ -0,0 +1,29 @@
+from flask import jsonify, Blueprint
+from flask import Flask, request, json, jsonify, make_response
+from database.models import user, receipt, picture
+from config import SENDGRID_API_KEY
+import os
+from sendgrid import SendGridAPIClient
+from sendgrid.helpers.mail import Mail
+
+email_handler = Blueprint('email_handler', __name__)
+
+
+@email_handler.route('/sendEmail', methods=['GET'])
+def sendEmail():
+
+    message = Mail(
+        from_email='teamravioli6@gmail.com',
```

Comment:
you can set this as a environment variable as well, so that it becomes easier to change in the future

Reply:
okay, will do that changes thanks!